src/main/java/org/wiztools/xsdgen/XsdGen.java

```diff
@@ -1,57 +1,49 @@
 package org.wiztools.xsdgen;
 
+import nu.xom.*;
+import org.apache.log4j.Logger;
+import org.wiztools.commons.Charsets;
+import org.wiztools.commons.StringUtil;
+
 import java.io.File;
 import java.io.IOException;
 import java.io.OutputStream;
 import java.nio.charset.Charset;
 import java.util.HashSet;
 import java.util.Set;
-import nu.xom.Attribute;
-import nu.xom.Builder;
-import nu.xom.Document;
-import nu.xom.Element;
-import nu.xom.Elements;
-import nu.xom.ParsingException;
-import nu.xom.Serializer;
-import org.wiztools.commons.Charsets;
-import org.wiztools.commons.StringUtil;
 
 /**
- *
  * @author subWiz
  */
 public final class XsdGen {
 
+    private static final Logger LOGGER = Logger.getLogger(XsdGen.class);
+
     private static final String XSD_NS_URI = "http://www.w3.org/2001/XMLSchema";
     private final String XSD_PREFIX;
 
     private Document doc = null; // populated after the parse method is called!
 
-    XsdGen(final String prefix) {
+    // @jardlabs maxOccurs support deactivated - should only be inferred unless having all element occurrences
+    private static final boolean MAX_OCCURS_ACTIVATED = false;
+
+    public XsdGen(final String prefix) {
         XSD_PREFIX = prefix;
     }
 
-    XsdGen() {
-        XSD_PREFIX = "xsd"; // Default XSD Prefix
+    public XsdGen() {
+        XSD_PREFIX = "xs"; // Default XSD Prefix
```

Comment:
Hi Subhash,

You’re right !  I’ve just updated code to ensure backward-compatibility.
Eric

On 06 May 2015, at 09:59, Subhash Chandran <notifications@github.com> wrote:

> In src/main/java/org/wiztools/xsdgen/XsdGen.java:
> 
> >          XSD_PREFIX = prefix;
> >      }
> >  
> > -    XsdGen() {
> > -        XSD_PREFIX = "xsd"; // Default XSD Prefix
> > +    public XsdGen() {
> > +        XSD_PREFIX = "xs"; // Default XSD Prefix
> Could possibly break backward-compatibility of the tool for existing users.
> 
> —
> Reply to this email directly or view it on GitHub.
>